sources/core/Stride.Core/Collections/Dequeue.cs

```diff
@@ -157,6 +169,30 @@ public int IndexOf(T item)
         return -1;
     }
 
+    public int BinarySearch(T item, IComparer<T>? comparer = null)
```

Comment:
It would be nice to document every public API.

Reply:
Done